### ISSUE TITLE
Switch to Fastly to avoid slow upgrades and connection issues

### ISSUE THREAD

**Issue by clefebvre** [linuxmint/mintupgrade#87]
Shortly after the announcements or during high load, the main repositories can get really busy.

This can result in a very slow upgrade, or even in connection issues and failures.

To avoid this, switch to the Fastly repositories. They are specifically designed for high load and high availability.

Open a terminal and type the following command to edit the sources list:

`sudo xed /etc/apt/sources.list.d/official-package-repositories.list`

In this file, find the line which contains `virginia main upstream import backport` and replace it with the line below:

`deb https://fastly.linuxmint.io virginia main upstream import backport`

Save the file with `Ctrl+S` and close the text editor.

Update the APT cache with the command below:

`apt update` 

Make sure there are no errors in the output.

That's it, your system is now using the Fastly repositories. You can now perform the upgrade without worrying about high server load.

**Comment by ScubaMan1970** [comment]
Thank you clefebvre. _MintOS continues to breathe new life into my geriatric Deskpro 6200_.

**Comment by renatogfilhinho** [comment]
Thank you!

**Comment by iyousif** [comment]
Thank you. Doing the upgarde now aws..

**Comment by elidoz** [comment]
thanks! should I change it back after the update?

**Comment by DutchPete** [comment]
> In this file, find the line which contains `virginia main upstream import backport`

This does not occur in the equivalent LMDE file. However, is it possible to just add `deb https://fastly.linuxmint.io virginia main upstream import backport` to my LMDE file?

**Comment by Dewillim** [comment]
So cool that the Team is testing new solutions! Thanks a lot, @clefebvre !

**Comment by atarax42** [comment]
I applied the change to the soures list, `apt update` shows no errors. But when I open `mintupdate` I get a notification that says, I should switch to another mirror server and https://fastly.linuxmint.io cannot be connected to.
Edit: `ping fastly.linuxmint.io` shows no errors.

**Comment by DutchPete** [comment]
@atarax42 I got the exact same thing yesterday. Until then I had the software manager set up for Fastly and it worked well.

So now I wonder if LMDE'ers can use the Linux Mint Fastly site @clefebvre mentions in the OP.

**Comment by Twixtfanatic** [comment]
I inadvertently deleted line 2 when I deleted the old line 1. Line 2 began with a # symbol. Can anyone tell me what it should be? BTW if I had used emacs instead of xed it would have saved the old file for me. Thank you.

**Comment by TheLinuxITGuy** [comment]
> I inadvertently deleted line 2 when I deleted the old line 1. Line 2 began with a # symbol. Can anyone tell me what it should be? BTW if I had used emacs instead of xed it would have saved the old file for me. Thank you.


Here's what mine looks like on LM 21.3:
                   
```
# Do not edit this file manually, use Software Sources instead.

deb http://packages.linuxmint.com virginia main upstream import backport #id:linuxmint_main

deb http://archive.ubuntu.com/ubuntu jammy main restricted universe multiverse
deb http://archive.ubuntu.com/ubuntu jammy-updates main restricted universe multiverse
deb http://archive.ubuntu.com/ubuntu jammy-backports main restricted universe multiverse

deb http://security.ubuntu.com/ubuntu/ jammy-security main restricted universe multiverse
```

**Comment by Twixtfanatic** [comment]
Thank you. Fortunately I took a snapshot with Timeshift before I started. FYI here is the line I am supposed to replace:

`deb http://packages.linuxmint.com virginia main upstream import backport #id:linuxmint_main`

Note the #id:linuxmint main at the end, which I believe I need to preserve when I edit.

**Comment by commander-keen** [comment]
> I applied the change to the soures list, `apt update` shows no errors. But when I open `mintupdate` I get a notification that says, I should switch to another mirror server and https://fastly.linuxmint.io cannot be connected to. Edit: `ping fastly.linuxmint.io` shows no errors.

Same here on Linux Mint 21.3:
![2024-07-29_Aktualisierungsverwaltung_001](https://github.com/user-attachments/assets/87667879-7807-4d76-92e5-63bacfa44341)
@clefebvre Is there an additional step required maybe?

apt update looks fine to me  (Issue is the same with http and https):

> OK:1 http://ppa.launchpad.net/cdemu/ppa/ubuntu jammy InRelease
> OK:2 https://repo.steampowered.com/steam stable InRelease                      
> OK:3 http://ppa.launchpad.net/mkusb/ppa/ubuntu jammy InRelease                 
> OK:4 https://dl.winehq.org/wine-builds/ubuntu focal InRelease                  
> Holen:5 http://ftp.hosteurope.de/mirror/archive.ubuntu.com jammy InRelease [270 kB]
> OK:6 http://ppa.launchpad.net/ubuntuhandbook1/avidemux/ubuntu jammy InRelease  
> Holen:7 https://mega.nz/linux/repo/xUbuntu_22.04 ./ InRelease [2.961 B]        
> OK:8 http://security.ubuntu.com/ubuntu jammy-security InRelease                
> OK:9 https://packages.microsoft.com/repos/ms-teams stable InRelease            
> Holen:10 https://download.mono-project.com/repo/ubuntu vs-bionic InRelease [5.144 B]
> OK:11 https://ppa.launchpadcontent.net/cappelikan/ppa/ubuntu jammy InRelease   
> OK:12 https://ppa.launchpadcontent.net/dupeguru/ppa/ubuntu jammy InRelease     
> OK:13 https://ppa.launchpadcontent.net/misery/ppa/ubuntu jammy InRelease       
> OK:14 https://ppa.launchpadcontent.net/strukturag/libde265/ubuntu jammy InRelease
> OK:15 https://ppa.launchpadcontent.net/tomtomtom/yt-dlp/ubuntu jammy InRelease 
> OK:16 https://ppa.launchpadcontent.net/xtradeb/apps/ubuntu jammy InRelease     
> OK:17 https://packages.teejeetech.com/battery-monitor/free/ubuntu/focal/amd64 stable InRelease
> OK:18 http://ftp.hosteurope.de/mirror/archive.ubuntu.com jammy-updates InRelease
> OK:19 http://ftp.hosteurope.de/mirror/archive.ubuntu.com jammy-backports InRelease
> Ign:20 http://fastly.linuxmint.io virginia InRelease                           
> OK:21 http://fastly.linuxmint.io virginia Release                              
> Ign:22 https://repo.vivaldi.com/stable/deb stable InRelease
> OK:23 https://repo.vivaldi.com/stable/deb stable Release
> Es wurden 278 kB in 6 s geholt (50,3 kB/s).
> Paketlisten werden gelesen… Fertig
> Abhängigkeitsbaum wird aufgebaut… Fertig
> Statusinformationen werden eingelesen… Fertig
> Aktualisierung für 2 Pakete verfügbar. Führen Sie »apt list --upgradable« aus, um sie anzuzeigen.

The two packages being held back are  python3-update-manager and update-manager-core (but that should not affect the reachablility of the server anyway...)

launching mintUpdate 6.0.9 from command line:

> mintupdate
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:1379: DeprecationWarning: Gdk.threads_init is deprecated
  Gdk.threads_init()
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:1764: DeprecationWarning: Gdk.threads_enter is deprecated
  Gdk.threads_enter()
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:803: DeprecationWarning: Gdk.threads_leave is deprecated
  Gdk.threads_leave()
Flatpak: done generating updates
(28, 'Resolving timed out after 5000 milliseconds')

...results in a timeout.

I now found https://github.com/linuxmint/linuxmint/issues/658#issuecomment-2048477853:

> python3 test.py
http://packages.linuxmint.com
 --> 1722155345
http://fastly.linuxmint.io
Traceback (most recent call last):
  File "/home/martin/Downloads/test.py", line 13, in <module>
    c.perform()
pycurl.error: (28, 'Resolving timed out after 5000 milliseconds')

I don't get it, because "ping fastly.linuxmint.io" shows no errors:

> ping fastly.linuxmint.io
PING t.sni.global.fastly.net (146.75.119.52) 56(84) bytes of data.
64 bytes from 146.75.119.52 (146.75.119.52): icmp_seq=1 ttl=59 time=30.3 ms
64 bytes from 146.75.119.52 (146.75.119.52): icmp_seq=2 ttl=59 time=30.3 ms
64 bytes from 146.75.119.52 (146.75.119.52): icmp_seq=3 ttl=59 time=33.3 ms
64 bytes from 146.75.119.52 (146.75.119.52): icmp_seq=4 ttl=59 time=31.0 ms
64 bytes from 146.75.119.52 (146.75.119.52): icmp_seq=5 ttl=59 time=29.8 ms
64 bytes from 146.75.119.52 (146.75.119.52): icmp_seq=6 ttl=59 time=23.4 ms

Altough the first start/ping takes roughly 4s or so, pretty close to 5000ms timeout maybe...?

**Comment by ssusbauer** [comment]
>  This does not occur in the equivalent LMDE file. However, is it possible to just add `deb https://fastly.linuxmint.io virginia main upstream import backport` to my LMDE file?

@DutchPete: The LMDE release is Faye, not Virginia. Using the wrong one could potentially break things. `deb https://fastly.linuxmint.io faye main upstream import backport` works fine for LMDE. Cinnamon has some big updates available.

LMDE is separate from Mint and does its own separate releases though. You are still going to be on LMDE 6, not Linux Mint 22. Wanted to make sure that's clear, a new Linux Mint version usually just means some package updates on LMDE.

**Comment by DutchPete** [comment]
@ssusbauer: Many thanks for your reply. So, this is what my `official-package-repositories.list` looks like:

```
deb https://fastly.linuxmint.io faye main upstream import backport 

deb https://deb.debian.org/debian bookworm main contrib non-free non-free-firmware
deb https://deb.debian.org/debian bookworm-updates main contrib non-free non-free-firmware
deb http://security.debian.org/ bookworm-security main contrib non-free non-free-firmware

deb https://deb.debian.org/debian bookworm-backports main contrib non-free non-free-firmware
```
with the one for Faye at the top. May I assume the list is now OK as it is?

BTW, yes, I am aware LMDE is separate from Linux Mint and that I will still be on LMDE 6.

I also got this output in the terminal:

```
user1@lenovo:~$ sudo xed /etc/apt/sources.list.d/official-package-repositories.list
[sudo] password for user1:             

** (xed:76941): WARNING **: 18:11:58.136: Set document metadata failed: Setting attribute metadata::xed-spell-language not supported
```
Is that something I need to worry about? It does not seem so to me.

**Comment by ssusbauer** [comment]
@DutchPete yes that one looks reasonable.

**Comment by DutchPete** [comment]
@ssusbauer Thank you. And the terminal output is nothing to worry about?

**Comment by ssusbauer** [comment]
@DutchPete no, it is truly a just a warning that the file type does not support that spelling metadata. Nothing to worry about.

**Comment by DutchPete** [comment]
@ssusbauer thanks for your help and patience!
Have a good day.

**Comment by ErikDB87** [comment]
Should we change this line again after upgrading? Or leave it at `fastly.linuxmint.io`?

**Comment by NeatNit** [comment]
Thank you! I did this based on the blog instructions and the upgrade went smoothly.

I second the question: is it safe to leave it at `fastly.linuxmint.io`?

I also would like to know, would Fastly be added to the official mirrors list in Software Sources? Currently it's not in the list.

![image](https://github.com/user-attachments/assets/2fb60557-aa22-4b8e-89b2-dd1a206b24f5)

(In the screenshot: sorted alphabetically, there's no Fastly under F)

**Comment by johnsageeek** [comment]
> thanks! should I change it back after the update?

No. Watch for updates to the Fastly trial and Mint will give you the updated verdict.

**Comment by melroy89** [comment]
> Should we change this line again after upgrading? Or leave it at `fastly.linuxmint.io`?

It's safe to leave it at fastly, however just know you are part of a beta-test. Since Fastly is officially in beta according to Clem (since April this year): https://forums.linuxmint.com/viewtopic.php?t=417369

Related GitHub ticket: https://github.com/linuxmint/linuxmint/issues/658

The test is still on-going. Since the ticket above isn't closed yet, I believe the Mint team is still collecting data and looking at the pricing.

**Comment by philer** [comment]
I'm trying to upgrade LM 22 -> 22.1 right now and watching a agonizingly slow download (4 hours remaining).  
The thing is, I've already changed my software sources. I was on a local mirror, but the upgrade actually just changed the sources list back to `packages.linuxmint.com` for xia, disregarding the configured mirror. I tried both fastly.linuxmint.io and a local mirror.

See https://github.com/linuxmint/mintupdate/issues/835

**Comment by KES777** [comment]
https://linuxmint-user-guide.readthedocs.io/en/latest/upgrade-to-mint-22.html

**Comment by Ljuk67** [comment]
This completely messed up my laptop. I replaced the first line and then sudo apt update asked me if I want to remove and install a LOT of packages. Well, I thought, as it's recommended by the Linux team, it can't be too risky + I have a Timeshift snapshot. 

how wrong was I. After reboot, nothing starts - I get into grub, and no matter what I select (Start Linux..., Advanced Linux startup options...., recovery mode, memory test) - NOTHING runs, everything hangs. 

I try a live USB with the same version linux that was running smoothly on my Clevo - same thing - anything beyond grub is stuck.

I am not a rookie, I fixed several boot issues on several laptops before, so I assumed - worst case scenario, I boot from a live usb and restore a Timeshift snapshot.

But now I'm stuck. 

BTW, could this be the reason?:
In  /etc/apt/sources.list.d/official-package-repositories.list

I REMOVED this first line
deb http://packages.linuxmint.com/ **wilma** main upstream import backport #id:linuxmint_main

/*note that the tutorial has **virginia** main upstream.... not **wilma**. does this mean that I tried to update the wrong version?*/

and ADDED this instead:
deb https://fastly.linuxmint.io/ virginia main upstream import backport #id:linuxmint_main

Anyway, posting here as a warning for others. It's not a straightforward update - and I have a Clevo linux based laptop, not some DIY dual boot on an old laptop.

**Comment by ssusbauer** [comment]
> BTW, could this be the reason?:

Yes, that is the reason. The only difference should be the URL, Linux Mint Virginia is 21.3. The system dramatically downgraded - but quickly!

That is a good note about the initial post, however. It’s from July 2024 when it worked as written, but needs modification for different releases. For a normal up to date system, there should be few if any package changes associated with this, as the mirrors should be nearly identical.

You’re not the only one who was confused by this: https://github.com/linuxmint/mintupgrade/issues/87#issuecomment-2256177881

**Comment by Jeremy7701** [comment]
Why did you make this change?

virginia is Mint 21.3
wilma is Mint 22.0
xia is Mint 22.1

Your change told Mint that your Mint was running 21.3 instead of the actual 22.0 it was running.
So of course it tried to remove many packages in an attempt to go back in time.

See [https://www.linuxmint.com/download_all.php](https://www.linuxmint.com/download_all.php) for the names.

**Comment by Ljuk67** [comment]
all my system info told me I'm running 21.3, believe me, i checked. My grub also shows 21.3. And yes, 21.3 in the system info, and wilma in the file. 

Maybe because i attempted an upgrade a year ago, but stopped it early?

I also restored a Timeshift  snapshot around that time (a year ago, i believe, not 100%sure), could that mix up the versions info?

Well, point is that such way of doing updates and giving users advanced and potentially dangerous instructions without any warnings, is not how a distro should be updated. 

Bad communication about how to update plus an  unsafe update, even if i took some missteps too. Hate to say, but Mint major updates are not user friendly, even if the user is advanced enough. 

And leaving outdated info that breaks things on the main update blog is really bad. 

Will probably have to figure out how to flash my bios with the laptop supplier. And stay away from future updates, too paranoid now 🤷.

**Comment by Jeremy7701** [comment]
You could do a fresh install from a download of 22.1 on USB.
Followed by a restore of your user data from your backup.

If you have no backup...
It should be possible to use a live boot to get a copy of important user data, provided you do this before any fresh installs to disk.

**Comment by Ljuk67** [comment]
That was my first step, but as I write above, I can't launch the live USB either. It gets stuck in the same way as my installed Linux.

So the damage isn't just Linux, but something before that, which is beyond my skills.

Might try a different distro, just to see if it launches at all.

**Comment by DutchPete** [comment]
@Ljuk67 if you are going to stay away from future updates  there is no point in staying with Mint, you might as well install a different distro. There is plenty of choice, although the large majority will not suit you, 1 or 2 might.

2 days ago I installed an app called virt-manager, which is a desktop graphical tool for managing virtual machines. It is very easy to install and it really blew me away: you can install any distro you like and it is as if it is a bare metal install.

It detects your wifi, so you can go online, download and install you favorite apps, do system updates, the lot. And you are not limited to just installing 1 distro in virt-manager.

I am using Endeavour OS, and [here](https://discovery.endeavouros.com/applications/how-to-install-virt-manager-complete-edition/2021/09/) is an article how to install virt-manager. The guideline are terminal-based and are for linux generally. The only thing you have to is to replace the Arch terms, such as `pacman` or `Syu` by the Mint/Ubuntu equivalents.

**Comment by Ljuk67** [comment]
I disagree, if i get it to work, mint is pretty good, even the older versions. Only the major version update is a pain. Or did you mean something else?

But thanks for the tip, virt-manager sounds good, once i get anything to run on my laptop,i'll try it out.  

I had Mint preinstalled, but now its all fecked up anyway, i might try Ubuntu. 

btw, i tried a Lubuntu live usb, same issue.

**Comment by DutchPete** [comment]
@Ljuk67 I did not say Mint is no good. But since you had already indicated you might move away from it, I merely reinforced your sentiment that staying with Mint if it is messed up on your system is pointless.

I tried Mint several times and was happy, until I wasn't. I got myself an ultramodern laptop a few weeks ago, and when I installed Mint on it, it did not work, black screen. So, I ditched it for Endeavour OS, the Cinnamon version.

On my 5 year old laptop I have LMDE and that works fine, it is solid and even though it is semi-rolling there are no hiccups.

**Comment by Ljuk67** [comment]
OK, that makes sense. In my case, i need a stable and as low hassle as possible distro (I don't want to look for drivers through forums etc) that can run most standard windows software. Mint fit the bill years ago when I did the research, was supposed to be the most stable too.

Wil see how this works out, hopefully my laptop supplier can help, they were really good so far.

**Comment by DutchPete** [comment]
@Ljuk67 I can recommend Endeavour. It is Arch-based, therefore a rolling release, but the upgrades are tested well before their release so the distro is pretty stable.

That does not mean there is no hitch occasionally, but the same can be said for any fixed release distro.

Another one that is very stable is openSUSE Leap.

**Comment by Ljuk67** [comment]
Those are all fringe distros, what's wrong with Ubuntu? 

Will check out both, thanks.

**Comment by DutchPete** [comment]
@Ljuk67 fringe distros? I beg to differ. In any case, you non-fringe distro Mint did not quite work out, did it?

Ubuntu is a Microsoft style OS whereby your opinion does not count and things are forced down your throat. Apart from that, Ubuntu is great.

**Comment by Ljuk67** [comment]
Compared to Ubuntu i mean. 

Im happy with a standard distro, not too much off the beaten path, but not Windows. The less software is built for that distro and the more DIY it is, the less appealing it isto me. Mint is still perfect, except the weird half DIY major updates. Guessing Ubuntu doesn't have that.

**Comment by DutchPete** [comment]
@Ljuk67 good to have expounded your condition. In that case EOS is not for you, but perhaps Zorin is? It is a popular distro, esp. among those switching from Windows.

Pop!_OS is also quite popular among the switchers, although Zorin is more so. Bear in mind that a popular concept re software is the so-called flatpaks. They are apps wrapped within their own containers carrying all the dependencies too, and there is no interference with dependencies in the system.

They are similar to Ubuntu's Snaps, although those are almost forced down your throat.

If you are interested in other distros, here is a site that lists all the distros and all their meta-data: [Distrowatch](https://distrowatch.com/).

**Comment by Ljuk67** [comment]
Quick update:

I had to flash my bios to get my laptop to work again. Thankfully, guys at laptopwithlinux.com are awesome and helped.

Now - why would an OS update fiddle about with my laptop firmware? It's supposed to operate on a lower level - so that if anything happens, I can still boot my laptop and try another system. But in this case, running this update messed up my BIOS. Not grub, BIOS. I don't like that. Happy for an OS to mess up it's own installation, but not laptop firmware/BIOS.

**Comment by DutchPete** [comment]
We've gone off topic.
A couple of weeks ago I received a TongFang laptop from *Laptop met Linux* and am very happy with it. Peter Pont is very good.

Are you sure it is the OS that messed with your firmware, instead of the firmware simply being out of date and the flashing updated it? The reason I ask is because last week I saw a comment from someone whose *Laptop met Linux* Clevo laptop also needed to have his Bios flashed to get it updated.

 I am way out of my depth to discuss this sensibly.

**Comment by Ljuk67** [comment]
Yeah it was Peter Pont, good guy. 

It's a 1 year old laptop, bought new, and the bios was updated once to fix another thing. It was working fine for a year.

As I wrote initially, I messed up too, so this started as an user (mine) error. That's not my issue here, but the fact an OS update shouldn't mess up my BIOS, so that i can't even launch from a live usb.

Let's stop here,  we made this thread much longer than needed anyway...

**Comment by Ljuk67** [comment]
Quick update: I ran the update process again, without this bit on the update page:

**_Warning! We are anticipating very high server load and slow download/connection speeds. We recommend to follow this guide to switch to Fastly prior to performing this upgrade: [Switch to Fastly to avoid slow upgrades and connection issues](https://github.com/linuxmint/mintupgrade/issues/87)._**

And it went fine.

That bit should never have been on the main update page in my opinion. It's misleading as it can lead to too many variables that aren't mentioned there, like in my case. I'm surprised - it's still there actually. People aren't updating only when you release the update - they can do that years later.

**Comment by bernd-wechner** [comment]
That's a great tip, but I wonder why we don't just make this switch as easy to remember as my birthday and to do as falling off a log. That is, maybe, put on the Software Sources dialog a simple option "Use fastly" ?

**Comment by Condobloke** [comment]
Has this progressed at all?

The suggestion above surely has merit....to include Fastly in Software Sources.

**Comment by NeatNit** [comment]
> Has this progressed at all?
> 
> The suggestion above surely has merit....to include Fastly in Software Sources.

I'm confused about this comment, because this was already done, wasn't it? The mirror called "Linux Mint" is Fastly. The old default mirror has been removed from the list.

<img width="624" height="432" alt="Image" src="https://github.com/user-attachments/assets/1d825da2-f493-4b1a-babb-2061773dad3b" />

Edit: https://github.com/linuxmint/linuxmint/issues/658#issuecomment-3679321079

**Comment by Condobloke** [comment]
if that in the creenshot is correct , that is news to me. 

There is nothing pertaining to that on the page. 

I am not saying you are wrong !...I am saying I have no idea whether 'Linux Mint' is fastly in disguise or not.

Why would they Not announce that change on this page ?

I could never find fastly as a mirror......I used the 'https://fastly.linuxmint.io' from the top of the page and assumed that was how it was to be used.

I have to admit I had a habit of skimming the page above....I could have very easily missed it, if in fact it is written there.

**Comment by NeatNit** [comment]
I don't know if it was announced "properly". The comment I linked to mentioned it 3 weeks ago (in a different issue). 

> As of version `1.2.9` of `mint-mirrors`, **fastly.linuxmint.io** has replaced **packages.linuxmint.com**.

You're right, communications about this could be a lot better.

**Comment by Condobloke** [comment]
""I also would like to know, would Fastly be added to the official mirrors list in Software Sources? Currently it's not in the list.

(In the screenshot: sorted alphabetically, there's no Fastly under F)""

This was said/printed some months back......there is nothing since then.

I think someone has just 'plucked' what they would like to see happen and stated it as fact !!

The practise of sources changing themselves without user intervention was happening back in January 2025(?)

Back to the grindstone.

With some sort of Irish luck, Clem will sail in and make sense of/sort out the uncertainty

**Comment by NeatNit** [comment]
I am certain of what I said, if you mouse over Linux Mint it shows you the source URL:

<img width="609" height="201" alt="Image" src="https://github.com/user-attachments/assets/9a7b5a9d-588c-4ba1-b522-ed164a66263e" />

Not to mention that when you select it you see the Fastly URL in Software Sources:

<img width="361" height="118" alt="Image" src="https://github.com/user-attachments/assets/73fc1391-6b04-40c9-ab57-a59e4bfe32b0" />

**Comment by Condobloke** [comment]
My friend, I owe you a debt of gratitude. I truly did not know that.

**Comment by Condobloke** [comment]
Done:

<img width="943" height="545" alt="Image" src="https://github.com/user-attachments/assets/62284d3f-8b30-4b98-afee-31df0c201a4f" />

**Comment by Condobloke** [comment]
Interestingly, linux/fastly, is as slow as a wet week

<img width="502" height="418" alt="Image" src="https://github.com/user-attachments/assets/6bfd3a2e-bdfe-4328-9217-1e92cf2d6436" />

AAR net is in Australia...aarnet.edu.au  (An educational institution)
